app/representers/api/v1/application_user_application_representer.rb

```diff
@@ -15,10 +16,19 @@ class ApplicationRepresenter < Roar::Decorator
              type: String,
              readable: true,
              writeable: false,
+             getter: ->(*) { application.name },
              schema_info: {
                required: true,
                description: "The name of this Application"
              }
 
+    collection :roles,
```

Comment:
Just a comment, not needing a change - but any reason to restrict the roles to things like `admin`, `editor`, etc? Just making sure we're okay with anything being in the role field.

Reply:
hm mainly the card didn't ask for any restrictions and I imagine the roles could be different for each app